src/app/content/components/Page/highlightManager.ts

```diff
@@ -137,11 +138,11 @@ export default (container: HTMLElement, getProp: () => HighlightProp) => {
           highlighter: listHighlighter,
           highlights: listPendingHighlight
             ? [
-                ...listHighlights.filter(
-                  (highlight) => !listPendingHighlight || highlight.id !== listPendingHighlight.id
-                ),
-                listPendingHighlight,
-              ]
+              ...listHighlights.filter(
+                (highlight) => !listPendingHighlight || highlight.id !== listPendingHighlight.id
+              ),
+              listPendingHighlight,
+            ]
```

Comment:
please avoid changing unrelated whitespace, it makes the diff harder to read, if it makes your life easier you can try using the `git add -p` option that allows you to stage chunks of diffs instead of whole files